Keas.Mvc/Controllers/TeamAdminController.cs

```diff
@@ -1090,6 +1095,26 @@ private Equipment CreateEquipment(EquipmentImport r, Team team, EquipmentImportR
                         ModelState.AddModelError("Type", "Invalid Equipment Type");
                     }
                 }
+
+                if (typesWithPartectionAndAvailability.Contains(equipment.Type))
+                {
+                    if (!string.IsNullOrWhiteSpace(r.ProtectionLevel) && protectionLevels.Contains(r.ProtectionLevel.Trim(), StringComparer.OrdinalIgnoreCase))
+                    {
+                        equipment.ProtectionLevel = protectionLevels.Single(a => a.Equals(r.ProtectionLevel.Trim(), StringComparison.OrdinalIgnoreCase));
+                    }
+                    else
+                    {
+                        equipment.ProtectionLevel = "P1";
```

Comment:
What do you think about defaulting to null, which could be an option in the select list equivalent to "unknown"?  Or we could even make a new status UK which is unknown.  Just want to be careful, not sure if defaulting to P1/A1 is right or not

Reply:
@srkirkland With my Script I'm defaulting everything that matches to P1 A1.
If we go with an "unknown" that basically means they can change it to unknown.

Reply:
I could have that record error out if you want.

Reply:
Yea I think changing to unknown might be a good option for people to have.  But for now let's default to P1/A1 and make the selection mandatory and see if that works.  Since it is required for IS3 maybe that's right anyway.  We'll discuss with adam when he comes back too.